### PR TITLE
Updated Support index.md

### PR DESCRIPTION
**Pull request by FinnianDempsey**
Added info about Priority support and when to contact sales along with free tier limits

### CODE REVIEW

**Comment by FinnianDempsey** [comment]
@steve-fenton-octopus & @tonykelly-octopus

File: src/pages/docs/support/index.md [hunk 1]
```diff
@@ -9,6 +9,13 @@ description: Information explaining how to perform requested tasks by Octopus su
 navOrder: 160
 ---
 
+If you want to enquire about purchasing, renewing or upgrading your Octopus license, please contact our [sales](https://octopus.com/company/contact) and [customer success](customersuccess@octopus.com) teams. For specific issues about the product please contact our support team.
```
Comment (TeganAli) [inline src/pages/docs/support/index.md:12]:
```suggestion
To enquire about purchasing, renewing, or upgrading your Octopus license, please [contact our sales team](https://octopus.com/company/contact). 

If you need product help and have a paid license, or you're trialing Octopus, please [email our support team](mailto:support@octopus.com). You can also visit our [Support page](https://octopus.com/support).
```

File: src/pages/docs/support/index.md [hunk 2]
```diff
@@ -9,6 +9,13 @@ description: Information explaining how to perform requested tasks by Octopus su
...
+
+For customers with a paid license, or if you're currently trialing Octopus, please reach out for support at [support@octopus.com](support@octopus.com). Otherwise please visit our [Support page](https://octopus.com/support).
```
Comment (TeganAli) [inline src/pages/docs/support/index.md:14]:
```suggestion
If you need product help and have a paid license, or you're trialing Octopus, please [email our support team](mailto:support@octopus.com). You can also visit our [Support page](https://octopus.com/support).
```

File: src/pages/docs/support/index.md [hunk 3]
```diff
@@ -9,6 +9,13 @@ description: Information explaining how to perform requested tasks by Octopus su
...
+
 Sometimes when you contact support we might ask you to perform tasks on your Octopus. This section explains how to perform some of those tasks.
```
Comment (TeganAli) [inline src/pages/docs/support/index.md:16]:
```suggestion
Sometimes when you contact support, we might ask you to perform tasks in Octopus. This section explains how to perform some of those tasks.
```

File: src/pages/docs/support/index.md [hunk 4]
```diff
@@ -9,6 +9,13 @@ description: Information explaining how to perform requested tasks by Octopus su
...
 
-To contact Octopus support, please visit the [support page](https://octopus.com/support).
+:::div{.hint}
+Priority Support is available as an addition to your Octopus Enterprise license. For more information on what is included, please see our [Priority Support page](https://octopus.com/support/priority).
```
Comment (TeganAli) [inline src/pages/docs/support/index.md:19]:
```suggestion
Priority Support is available as an addition to your Octopus Enterprise license. For more information, please see our [Priority Support page](https://octopus.com/support/priority).
```

**Comment by TeganAli** [comment]
@FinnianDempsey do you want to accept these suggestions if you're happy with them? Thank you :)

**Review by TeganAli — APPROVED** [review]

**Pull request merged**